### PR TITLE
Design cleanup

### PR DESCRIPTION
**Pull request by Kriesse**
Removed the dat-design dep, cleaned up messy styles, started to use tachyons.
This should help to make the transition to sheetify/base-elements easier and avoid breaking all the styles.

### CODE REVIEW

File: pages/main.js [hunk 1]
```diff
@@ -95,27 +91,27 @@ function createTable (dats, send) {
...
             text: '',
-            klass: 'row-action',
+            cls: 'row-action',
             click: () => send('app:open', dat)
           })}
           ${button({
             icon: 'link',
             text: '',
-            klass: 'row-action',
+            cls: 'row-action',
```
Comment (yoshuawuyts) [inline pages/main.js]: these `cls` properties can safely be named `class` which might make them a bit easier to understand

File: scss/imports/_header.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+.menu-trigger {
+  height: 2rem;
+  padding-left: .625rem;
+  width: 2.5rem;
+  .icon-menu {
+    fill: $color-neutral--20;
+  }
+  &:hover, &:focus {
+    .icon-menu {
```
Comment (yoshuawuyts) [inline scss/imports/_header.scss]: I do find these statements a bit hard to read, I think this translates to:
```css
.menu-trigger:hover .icon-menu,
.menu-trigger:focus .icon-menu {}
```
right? Just wanna make sure I understand what's going on alright haha

File: pages/main.js (hunk 1 above)
Comment (juliangruber) [inline pages/main.js]: doesn't that mess with anything that tries to parse the source? `class` is a reserved keyword

File: app/public/css/main.css [hunk 3]
```diff
@@ -0,0 +1,1734 @@
+/**
```
Comment (juliangruber) [inline app/public/css/main.css]: is this file hand written or generated?

**Comment by yoshuawuyts** [comment]
Nope it's fine, reserved keywords as property names were ok going ES5
onward iirc

**Comment by juliangruber** [comment]
> Nope it's fine, reserved keywords as property names were ok going ES5
> onward iirc

Oh, learned something :) Ok if this is only ever going to be used in Dat Desktop, I'm +1, otherwise we do need to worry about browsers and old node versions.

**Comment by yoshuawuyts** [comment]
Oh yeah afaik this runs like IE7+ or something / all node versions haha -
like ES5 has been around for a while I think. I've never run into issues at
least (:

File: app/public/css/main.css (hunk 3 above)
Comment (Kriesse) [inline app/public/css/main.css]: @juliangruber it's generated – i guess the entire `app` directory should be gitignored? (where does that even come from? i'm confused)

File: scss/imports/_header.scss (hunk 2 above)
Comment (Kriesse) [inline scss/imports/_header.scss]: Yup, that's understood correctly. I love the `&` selector for this particular use case: I almost always use the same styles for `&:hover, &:focus` for easy accessibility wins and find the grouping helpful – but can see that it’s a bit hard to read if not used to it.
Can we have `&` in sheetify at all?

File: app/public/css/main.css (hunk 3 above)
Comment (juliangruber) [inline app/public/css/main.css]: oh! it's a leftover of a change that was reverted already. you can safely remove it

**Comment by karissa** [comment]
OK I fixed the merge conflict and cleaned up the unneeded main.css file

**Pull request merged**